etc/bash_completion.d/dnf

```diff
@@ -20,21 +20,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 # 02110-1301  USA
 
-__dnf_commands="
-autoremove autoremove-n autoremove-na autoremove-nevra builddep build-dep check
-check-update check-upgrade clean config-manager copr debug-dump debuginfo-install
-debug-restore deplist distribution-synchronization distrosync distro-sync downgrade
-download erase erase-n erase-na erase-nevra fedup group grouperase groupinfo groupinstall
-grouplist groupremove groups groupupdate help history info info-sec info-security
-info-updateinfo install install-n install-na install-nevra leaves list list-sec
-list-security list-updateinfo localinstall makecache mark needs-restarting playground
-provides reinstall remove remove-n remove-na remove-nevra repoclosure repograph
-repo-graph repoinfo repolist repomanage repo-packages repo-pkgs repoquery repoquery-n
-repoquery-na repoquery-nevra repository-packages repository-pkgs reposync search shell
-summary-updateinfo swap system-upgrade update updateinfo update-minimal update-to
-upgrade upgrade-minimal upgrade-to versionlock whatprovides
-"
-
```

Comment:
I don't think this is supposed to be added in the previous commit and then removed here.